modules/storages/lib/api/v3/file_links/file_link_representer.rb

```diff
@@ -144,7 +145,7 @@ def _type
     private
 
     def user_allowed_to_manage?(model)
-      current_user.allowed_to?(:manage_file_links, model.container.project)
+      model.container && current_user.allowed_to?(:manage_file_links, model.project)
```

Comment:
minor: can you make that `nil` check here explicit? With 

```ruby
model.container.present?
```

Reply:
Yes, sure.